city_scrapers/spiders/example.py

```diff
@@ -0,0 +1,231 @@
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    start_urls = [
+        (
+            "https://www.stlouis-mo.gov/government/departments/public-service/index.cfm"
+        )
+    ]
+
+    def __init__(self, *args, **kwargs):
+        self.agenda_map = defaultdict(list)
+        super().__init__(*args, **kwargs)
+
+    def parse(self, response):
+        self._parse_links(response)
+        yield from self._parse_meetings_page(response)
+
+    def _parse_meetings_page(self, response):
+        urls = [
+            (
+                "https://www.stlouis-mo.gov/events/"
+                "past-meetings.cfm?span=-30&department=332"
+            ),
+            "https://www.stlouis-mo.gov/events/all-public-meetings.cfm?span=30",
+        ]
+        for url in urls:
+            yield scrapy.Request(
+                url=url, method="GET", callback=self._parse_events_page
+            )
+
+    def _parse_events_page(self, response):
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url, callback=self._parse_event, dont_filter=True)
+
+    def _get_event_urls(self, response):
+        event_urls = response.css("ul.list-group h4 a::attr(href)").getall()
+        event_sponsors = response.css("ul.list-group li span.small::text").getall()
+        urls = []
+        for url, sponsor in zip(event_urls, event_sponsors):
+            if "aldermen" in sponsor.lower() or "aldermanic" in sponsor.lower():
+                urls.append(response.urljoin(url))
+        return urls
+
+    def _parse_event(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        start = self._parse_start(response)
+        links_key = datetime.strftime(start, "%m-%d-%y")
+
+        meeting = Meeting(
+            title=self._parse_title(response),
+            description=self._parse_description(response),
+            classification=self._parse_classification(response),
+            start=start,
+            end=self._parse_end(response),
+            all_day=self._parse_all_day(response),
+            location=self._parse_location(response),
+            source=response.url,
+        )
+
+        if meeting["classification"] == BOARD:
+            if links_key in self.agenda_map.keys():
+                meeting["links"] = self.agenda_map[links_key]
+        else:
+            meeting["links"] = []
+
+        meeting["status"] = self._get_status(meeting)
+        meeting["id"] = self._get_id(meeting)
+        return meeting
+
+    def _parse_title(self, response):
+        """Parse or generate meeting title."""
+        title = response.css("div.page-title-row h1::text").get()
+        title = title.replace("Meeting", "").replace("Metting", "")
+        title = title.replace("-", "- ")
+        title = title.replace("(Canceled)", "Cancelled")
+        return title.replace("  ", " ").strip()
```

Comment:
It looks like the Board of Public Service's meeting titles are either `Board of Public Service` or `Special Board of Public Service Meeting`. So you can do something like [this](https://github.com/stl-public-meetings/city-scrapers-stl/blob/de32c9cc4827de7cd8f6ec6f97d6574e894cafb1/city_scrapers/spiders/stl_land_reutilization.py#L89-L105) for `_parse_title`.